_team/joshhorton.md

```diff
@@ -1,10 +1,12 @@
 ---
 name: Josh Horton
 position: Software Scientist
-image_path: https://avatars.githubusercontent.com/u/30259414?v=4
+image_path: https://media.licdn.com/dms/image/v2/C5603AQFbJ8CicIHvfQ/profile-displayphoto-shrink_200_200/profile-displayphoto-shrink_200_200/0/1517285245980?e=1736380800&v=beta&t=r9g4kGKxo62UvYvV42Yqf5mBgDp1XLGzpJYkEdlawzM
 github: jthorton
 scholar: citations?user=qmngJQsAAAAJ&hl=en
 blurb: >-
-  Josh joined OpenFE in October 2024.
+  Josh received his Ph.D from the University of Newcastle in the Cole group. His research is focused on developing 
```

Comment:
```suggestion
  Josh received his Ph.D from the University of Newcastle in the Cole group. His research was focused on developing 
```
do you want past tense here?

Reply:
How about removing the word altogether and going with "his research focused"

Reply:
Thanks!